src/pages/loading-waiting-states/progress-bar/index.tsx

```diff
@@ -0,0 +1,313 @@
+                            color={'inherit'}
+                            onClick={(): void => {
+                                dispatch({ type: TOGGLE_DRAWER, payload: true });
+                            }}
+                            edge={'start'}
+                            style={{ marginRight: 20 }}
+                        >
+                            <MenuIcon />
+                        </IconButton>
+                    </Hidden>
+                    <Typography variant={'h6'} color={'inherit'}>
+                        Progress Bars
+                    </Typography>
+                    <div />
+                </Toolbar>
+            </AppBar>
+            <div className={classes.container}>
+                <div className={classes.uploadButtonContainer}>
+                    <Button variant={'contained'} color={'primary'} startIcon={<Publish />} onClick={uploadFile}>
+                        UPLOAD NEW FILE
+                    </Button>
+                </div>
+                <Card>
+                    <CardContent className={classes.cardContent}>
+                        <FormControl className={classes.formControl} component="fieldset">
+                            <RadioGroup
+                                aria-label="folder"
+                                name="folder"
+                                value={radioButtonvalue}
+                                onChange={changeRadioGroup}
+                            >
+                                {foldersList.map((option, i) => (
+                                    <FormControlLabel
+                                        className={classes.formLabel}
+                                        key={i}
+                                        value={option.value}
+                                        control={<Radio />}
+                                        label={
+                                            <div className={classes.radioLabel}>
+                                                <Folder className={classes.icon} />
+                                                <Typography style={{ marginLeft: '16px' }}> {option.label} </Typography>
+                                            </div>
+                                        }
+                                    />
+                                ))}
+                            </RadioGroup>
+                        </FormControl>
+                    </CardContent>
+                </Card>
+                <List data-cy={'list-content'} disablePadding component="nav" className={classes.placementOfList}>
+                    {fileUploadList.map(
+                        (item, i): JSX.Element => (
+                            <div key={`itemKey${item.id}`} className={classes.infoList}>
+                                <Snackbar
+                                    classes={{ root: classes.snackbar }}
+                                    open={item.open}
```

Comment:
this should just be `true` right? Once an item is removed from the array it will no longer be rendered (i.e., we should have anything that's in the list but not rendered to the screen)?

Reply:
The open prop gives a better animation, instead of just a hard delete... but maybe that will make it overly complicated.